filepathfilter/filepathfilter.go

```diff
@@ -61,30 +63,20 @@ func (f *Filter) Allows(filename string) bool {
 		return true
 	}
 
-	if len(f.include)+len(f.exclude) == 0 {
-		return true
```

Comment:
Good question -- I removed this special case because it adds no behavior that the function doesn't already accomplish. Without any includes/excludes, both of the `for` loops below don't execute, and we fall through into the base case: `return true`. I am happy to remove code when possible, so that is the rationale behind this 😄.